lib/internal/streams/utils.js

```diff
@@ -13,6 +13,7 @@ const kIsReadable = Symbol('kIsReadable');
 const kIsDisturbed = Symbol('kIsDisturbed');
 
 const kIsClosedPromise = SymbolFor('nodejs.webstream.isClosedPromise');
+const kControllerErrorFunction = SymbolFor('nodejs.webstream.controllerErrorFunction');
```

Comment:
Why are we using `Symbol.for` here?

Reply:
I used the similar pattern to what we followed over here https://github.com/nodejs/node/pull/46205#discussion_r1070552627

Reply:
@aduh95 I presume Node.js streams or another part that exists both in core and userland added this for interoperability (so that `require("node:stream")` and `require("readable-stream")` interoperate) and then everyone (probably me too) saw the code and cargo culted. Probably a bunch at the point we moved private variables to symbols when privates were not supported in snapshots back then.

Reply:
It's also arguably easier to debug "from the outside" but not by a considerable margin.